Solutions/Votiro/Analytic Rules/VotiroFileBlockedFromConnector.yaml

```diff
@@ -0,0 +1,28 @@
+id: 17bf3780-ae0d-4cd9-a884-5df8b687f3f5
+name: Votiro - File Blocked from Connector
+description: |
+ 'If the syslog message includes-Blocked under  -Sanitization result-field  and false under -password protected- field and null under -from- field  create an alert with the following message:[file name] with hash [file hash] that was sent from connector [connector name] was blocked by Votiro due to Policy [policy name], see more detail in the following link [incident url]'
```

Comment:
Description should explain the intend of the rule in plain English instead of explaining each field level detail.

Reply:
hi @devikamehra,
We have changed as per requested.